src/ts/waterfall/details-overlay/overlay-manager.ts

```diff
@@ -22,7 +22,7 @@ export default class OverlayManager implements OverlayManagerClass {
   /**
```

Comment:
just renamed `accordionHeight` to `datailsHeight`, since we don't call it accordion anywhere else.

Perhaps we should find a better name than "details-overlay" and refactor it globally at some stage.